tests/test_setproject.py

```diff
@@ -0,0 +1,45 @@
+from pew._utils import invoke_pew as invoke
+from utils import skip_windows
```

Comment:
I'm a bit worried that this might work as a relative import under Python2, thus [possibly breaking py.test invocations](http://stackoverflow.com/a/10367612/293735)

I haven't tried to run the code yet... this is just a note for things I'll want to look at before merging